src/MahApps.Metro/MahApps.Metro.Shared/Controls/MetroWindow.cs

```diff
@@ -857,6 +877,19 @@ public MetroWindow()
         {
             DataContextChanged += MetroWindow_DataContextChanged;
             Loaded += this.MetroWindow_Loaded;
+
+            // BorderlessWindowBehavior initialization has to occur in constructor. Otherwise the load event is fired early and performance of the window is degraded.
+            this.InitializeBorderlessWindowBehavior();
+        }
+
+        /// <summary>
+        /// Initializes the BorderlessWindowBehavior which is needed to render the custom WindowChrome.
+        /// </summary>
+        private void InitializeBorderlessWindowBehavior()
+        {
+            var behavior = new BorderlessWindowBehavior();
+
+            Interaction.GetBehaviors(this).Add(behavior);
```

Comment:
same here: this should be `WindowChromeBehavior` instead of the old `BorderlessWindowBehavior`

also: all behaviors must be add here and not at the style

```
private void InitializeStylizedBehaviors()
{
    var collection = new StylizedBehaviorCollection();
    collection.Add(new WindowChromeBehavior());
    collection.Add(new WindowsSettingBehaviour());
    collection.Add(new GlowWindowBehavior());
    StylizedBehaviors.SetBehaviors(this, collection);
}
```

without that, the glow and settings behaviors are not working